README.md

```diff
@@ -96,6 +96,37 @@ You can also set the following mount points by passing the `-v /host:/container`
 directory has the appropriate permissions and that the owner and group of the directory
 matches the user UID or name which is running inside the container.**
 
+Inclusion of the `oc` binary
+---------------------------------
+
+To assist in interacting with the OpenShift API server while using this image, the `oc` binary, the CLI command for OpenShift, has been
+installed in this image.
+
+However, it needs to noted that backward compatibility is not guaranteed between different versions of `oc` and the OpenShift
+API Server.  As such, it is recommended that you align versions of this image present in the nodes of your cluster with your 
+OpenShift API server.  In other words, you should use the version specific tag instead of the `latest` tag.
+
+|  Jenkins image version      | `oc` client version      |
+| :-------------------------- | ------------------------ |
+|  `jenkins-*-centos7:v3.7`   | 3.7 `oc` binary          |
+|  `jenkins-*-centos7:v3.6`   | 3.6 `oc` binary          |
+|  `jenkins-*-centos7:latest` | 3.9 `oc` binary          |
```

Comment:
I think that if you state 3.9 here, it'll need to be changed next release.  Might be better to say "whatever the latest version is" (or words to that effect)?

Reply:
Sure ... it is literally "the version of oc present in the docker.io/openshift/origin:latest image at the time the jenkins images with the latest tag were built".

I'll work on some consumable form of that.  Suggestions are of course welcome.